src/cmdc_tools/datasets/official/PA/data.py

```diff
@@ -9,42 +9,41 @@
 class Pennsylvania(DatasetBaseNoDate, ArcGIS):
     ARCGIS_ID = "xtuWQvb2YQnp0z3F"
     source = (
-        "https://www.arcgis.com/apps/opsdashboard/"
-        "index.html#/85054b06472e4208b02285b8557f24cf"
+        "https://experience.arcgis.com/experience/" "cfb3803eb93d42f7ab1c2cfccca78bf7/"
     )
     state_fips = int(us.states.lookup("Pennsylvania").fips)
     has_fips = False
 
     def get(self):
         df = self.get_all_sheet_to_df(
-            service="County_Case_Data_Public", sheet=0, srvid=2
+            service="Pennsylvania_Public_COVID19_Dashboard_Data", sheet=0, srvid=2
         )
 
         column_map = {
-            "COUNTY_NAM": "county",
+            "County": "county",
             "Cases": "cases_total",
+            "Probable": "cases_suspected",
             "Deaths": "deaths_total",
-            "AvailableBedsAdultICU": "available_icu_beds",
-            "COVID19Hospitalized": "hospital_beds_in_use_covid_total",
-            "TotalVents": "ventilators_capacity_count",
-            "VentsInUse": "ventilators_in_use_any",
-            "COVID19onVents": "ventilators_in_use_covid_total",
+            "Confirmed": "positive_tests_total",
+            "Negative": "negative_tests_total",
+            "Adult_ICU_Beds_Available": "icu_beds_available",
+            "Adult_ICU_staffed": "icu_beds_capacity_count",
+            "Med_Surg_Beds_Available": "hospital_beds_available",
+            "Med_Surg_Staffed": "hospital_beds_capacity_count",
```

Comment:
I don't think we are being consistent on the meaning of `hospital_beds_\w+` throughout the database.

We need to take a stand on whether or not the `hospital_beds_\w+` variables are inclusive or exclusive of the ICU. I think this one shows exclusivity, but I believe other scrapers (and even reports we grab directly) maybe inclusive.

Reply:
@cc7768 what do you think about this?